tardis/montecarlo/struct.py

```diff
@@ -0,0 +1,105 @@
+from ctypes import *
```

Comment:
This is bad! you should never do a `from xxx import *` - you will likely pollute your namespace with it.